src/authorization.ts

```diff
@@ -6,15 +6,15 @@ const authContextHasChangemakerPermission = (
 	permission: Permission,
 ): boolean =>
 	auth.role.isAdministrator ||
-	!!auth.user.permissions.changemaker[changemakerId]?.includes(permission);
+	(auth.user.permissions.changemaker[changemakerId] ?? []).includes(permission);
```

Comment:
Let the record show that I had just rewritten these lines in a previous commit (but I think in this case the history is helpful since it shows exactly how these rules impacted the code base)